Day6/que3.js

```diff
@@ -0,0 +1,63 @@
+
+// 3.1 Create a class Person with properties name, age, gender, and interests. Add a method greeting() that returns a string introducing the person. Also add a method farewell() that returns a string saying goodbye to the person.
+// 3.2 Create a class Student that inherits from the Person class and has a property studies. Override the greeting() method to include information about what the student is studying.
```

Comment:
The question asks you not to ignore the info provided by parent greeting class, but to also include additional info alongside. Please make change and try to understand the questions going forward.

Reply:
While doing this, try and make use of parent class greeting function as well.

Reply:
![Uploading day6final.png…]()